aiohttp/web.py

```diff
@@ -230,7 +231,15 @@ def router(self):
     def middlewares(self):
         return self._middlewares
 
-    def make_handler(self, *, loop=None, **kwargs):
+    def make_handler(self, *,
+                     loop=None,
+                     access_log_class=AccessLogger,
+                     **kwargs):
+
+        if not issubclass(access_log_class, AbstractAccessLogger):
+            raise TypeError('access_log_class must be subclass of '
+                            'aiohttp.abc.AbstractAccessLogger')
```

Comment:
Would be nice to add `got {}` to error message to simplify understanding what's wrong with the logger.